components/net/image_cache_thread.rs

```diff
@@ -432,15 +393,11 @@ impl ImageCache {
                     Ok(()) => {
                         let pending_load = self.pending_loads.get_by_key_mut(&msg.key).unwrap();
                         pending_load.result = Some(result);
-                        let bytes = mem::replace(&mut pending_load.bytes, vec!());
+                        let bytes = pending_load.bytes.clone();
```

Comment:
It seems unfortunate to have to copy the entire image here, when it previously didn't. Is that required?